model-optimizer/extensions/front/tf/TFSliceToSlice.py

```diff
@@ -44,7 +44,8 @@ def replace_sub_graph(self, graph: Graph, match: dict):
 
         eq_node = Equal(graph, {'name': slice_name + '/equal'}).create_node()
         minus_one_node = Const(graph, {'name': slice_name + '/minus_one', 'value': np.array(-1)}).create_node()
-        int32_max_node = Const(graph, {'name': slice_name + '/int32_max', 'value': np.iinfo(np.int32).max}).create_node()
+        int32_max_node = Const(graph, {'name': slice_name + '/int32_max', 'value': np.int64(np.iinfo(np.int32).max)}).create_node()
```

Comment:
Instead of `np.int64(np.iinfo(np.int32).max)` use `int64_array(np.iinfo(np.int32).max)`.
Also, use `int64_array` method for the `minus_one_node`.

Also, please, add a comment in the code here describing why we insert constant of type int64 with max value of int32.

Reply:
Done